Ex1/test_ex1.py

```diff
@@ -53,6 +35,14 @@ def login(self, standard_user, secret_sauce):
         self.driver.find_element(*LoginLocators.user_name).send_keys(standard_user)
         self.driver.find_element(*LoginLocators.password).send_keys(secret_sauce)
         self.driver.find_element(*LoginLocators.login_button).click()
+        
+    def login_exists(self):
+        try:
+            button = self.driver.find_element(*LoginLocators.login_button).is_displayed()
+            if button is True:
+                return True
```

Comment:
this code is replicated in couple of places, maybe think of introducing util method outside classes like:

```
class Utils: 

def exists(driver,locator):
  try:
     element = driver.find_element(locator).is_displayed()
      if element is True:
         return True
  except NoSuchElementException:
       return False
```

and then use it like:
`Utils.exists(self.driver,*LoginLocators.login_button)`